backend/src/test/java/com/bakdata/conquery/integration/json/QueryTest.java

```diff
@@ -127,16 +133,12 @@ private void importPreviousQueries(StandaloneSupport support) throws JSONExcepti
 		}
 	}
 
-	private void importTableContents(StandaloneSupport support) throws IOException, JSONException {
-		CsvParserSettings settings = new CsvParserSettings();
-		CsvFormat format = new CsvFormat();
-		format.setLineSeparator("\n");
-		settings.setFormat(format);
-		settings.setHeaderExtractionEnabled(true);
+	public static void importTableContents(StandaloneSupport support, Collection<RequiredTable> tables, Dataset dataset) throws IOException, JSONException {
```

Comment:
👍 
Ich glaube es gibt eine IntegrationUtil Klasse, in die die static Funktionen verschoben werden können